mailpoet/assets/js/src/landingpage/faq.tsx

```diff
@@ -0,0 +1,128 @@
+import ReactStringReplace from 'react-string-replace';
+import { __ } from '@wordpress/i18n';
+import { Heading } from 'common/typography/heading/heading';
+
+function Faq() {
+  const list: {
+    slug: string;
+    title: string;
+    text: string;
+    readMoreText: string;
+    readMoreLink: string;
+  }[] = [
+    {
+      slug: 'item-1',
+      title: __(
+        'What types of campaigns can I create with MailPoet?',
+        'mailpoet',
+      ),
+      text: __(
+        'MailPoet allows you to create five different types of emails: Newsletter, Welcome Email, Latest Post Notifications, Re-engagement Emails and WooCommerce.',
```

Comment:
Wondering if you've verified `./do translations:build` exports these strings to `mailpoet.pot`?